Handling_Resources.php

```diff
@@ -0,0 +1,74 @@
+class Handling_Resources
+{
+  private $init_arr = [];
+  static $data = [];
+
+  function __construct($suppliers)
+  {
+    foreach ($suppliers as $supplier) {
+      $this->init_arr = [...$this->init_arr, ...json_decode(file_get_contents($supplier))];
+    }
+
+    Handling_Resources::$data = $this->formatting($this->init_arr, Handling_Resources::$data);
+    Handling_Resources::$data = $this->sorting(Handling_Resources::$data);
+
+    if (isset($_GET['format'])) {
+      $serveAPI = new Serve($_GET['format']);
+    }
+  }
+
+  private function replace_key($arr, $oldkey, $newkey)
+  {
+    $keys = array_keys((array) $arr);
+    if (array_key_exists($oldkey, (array) $arr)) {
+      $keys[array_search($oldkey, $keys)] = $newkey;
+    }
+    return $arr = array_combine($keys, (array) $arr);
+  }
+
+  private function formatting($init_arr, $arr)
+  {
+    foreach ($init_arr as $nested) {
+
+      $nested = (array) $nested;
+      if (strpos($nested['Rate'], '*') === 0) {
+        $nested['Rate'] = strlen($nested['Rate']);
+      }
+      $nested = $this->replace_key($nested, 'Fare', 'Price');
+      $nested = $this->replace_key($nested, 'Hotel', 'hotelName');
+      $nested = $this->replace_key($nested, 'amenities', 'roomAmenities');
+      $arr[] = $nested;
+    }
+    return $arr;
+  }
+
+  static function array_to_xml($data, $xml_data)
```

Comment:
You used "static" here, to be able to call it in the other class, Why didn't you use inheritance or composition.?
static is evil
https://stackoverflow.com/questions/7026507/why-are-static-variables-considered-evil#:~:text=Static%20variables%20are%20generally%20considered,assumptions%20of%20object%2Doriented%20programming.
https://breadcrumbscollector.tech/staticmethod-considered-a-code-smell/